integration/spark-common/src/main/scala/org/apache/carbondata/events/DataMapEvents.scala

```diff
@@ -60,7 +60,8 @@ case class BuildDataMapPreExecutionEvent(sparkSession: SparkSession,
  * example: bloom datamap, Lucene datamap
  */
 case class BuildDataMapPostExecutionEvent(sparkSession: SparkSession,
-    identifier: AbsoluteTableIdentifier, segmentIdList: Seq[String], isFromRebuild: Boolean)
+    identifier: AbsoluteTableIdentifier, segmentIdList: Seq[String],
+    isFromRebuild: Boolean, dmName: String)
```

Comment:
You can adjust the sequence of the parameters by moving `dmName` after `identifier`. This will make the method easy to understand: for some table's some datamap, for the corresponding segments doing rebuild or not